Filter/ModelFilter.php

```diff
@@ -134,4 +134,30 @@ protected function association(ProxyQueryInterface $queryBuilder, $data)
 
         return array($alias, false);
     }
+
+    /**
+     * Retrieve the parent alias for given alias.
+     * Root alias for direct association or entity joined alias for association depth >= 2.
```

Comment:
try to make it clear with some exemples : 

- if I filter on entity.relation.relation_field i'll get relation as parent alias
- if I filter on entity.relation.sub_relation.field i'll get sub_relation as parent alias
- . . .